selinux/ceph-installer.fc

```diff
@@ -0,0 +1,11 @@
+/usr/bin/gunicorn_pecan		--	gen_context(system_u:object_r:ceph_installer_exec_t,s0)
```

Comment:
Yeah, we should probably do that. This package does not ship the executables and we should not really enforce any rules on the executables outside the scope of the package itself.